dist/leaflet.css

```diff
@@ -330,6 +330,25 @@ svg.leaflet-image-layer.leaflet-interactive path {
 	border-bottom-right-radius: 2px;
 	}
 
+
+/* Dark Mode */
+.leaflet-darkmode .leaflet-bar a,
+.leaflet-darkmode .leaflet-bar a:hover{

```

Comment:
These styles apply to both the default state and on `:hover`, should probably remove `.leaflet-darkmode .leaflet-bar a:hover` here?